std/conv.d

```diff
@@ -4955,6 +4957,17 @@ unittest
     emplaceRef!int(i, 5);
 }
 
+@safe unittest
+{
+    static struct S
+    {
+        this(this) @safe {}
```

Comment:
Can you also write this unittest in such a way that you also test that unsafe postblit is still unsafe. It's so easy to just throw add some `@trusted` which isn't actually safe.

Could you also add a test for static arrays? Speaking of which, I *believe* the code in static arrays also does a typeid.postblit thing. Did you check if that code can be improved too?